cardano-cli/src/Cardano/CLI/EraBased/Options/Common.hs

```diff
@@ -3087,6 +3087,21 @@ pDRepActivity =
     , Opt.help "DRep activity period, in epochs."
     ]
 
+toNonNegativeIntOrErr :: Rational -> L.NonNegativeInterval
+toNonNegativeIntOrErr r =
+  case L.boundRational r of
+    Nothing -> error $ "toNonNegativeIntOrErr: rational out of bounds: " <> show r
+    Just n -> n
+
+-- Parser for MinFeeRefScriptCostPerByte
+pMinFeeRefScriptCostPerByte :: Parser L.NonNegativeInterval
+pMinFeeRefScriptCostPerByte =
+  Opt.option (toNonNegativeIntOrErr <$> readRational)  $ mconcat
+    [ Opt.long "min-fee-ref-script-cost-per-byte"
+    , Opt.metavar "RATIONAL"
+    , Opt.help "Reference scripts fee for the minimum fee calculation"
```

Comment:
"Reference script cost per byte for free calculation" and "ref-script-cost-per-byte" would be clearer.